codespell_lib/data/dictionary.txt

```diff
@@ -4006,20 +4009,42 @@ certaint->certain
 certaion->certain
 certan->certain
 certficate->certificate
+certficated->certificated
 certficates->certificates
+certfication->certification
 certian->certain
 certianly->certainly
 certicate->certificate
+certicated->certificated
 certicates->certificates
+certication->certification
 certifacte->certificate
+certifacted->certificated
 certifactes->certificates
+certifaction->certification
 certifcate->certificate
+certifcated->certificated
 certifcates->certificates
 certifcation->certification
+certifciate->certificate
+certifciated->certificated
+certifciates->certificates
+certifciation->certification
 certificat->certificate
+certificatd->certificated
+certificats->certificates
```

Comment:
I think you've missed `certificats->certification`

Reply:
`certificats->certification` doesn't look right for me, do you had something else in mind and was this just a copy'n'paste mistake?

Reply:
Indeed I did, I failed to edit it properly, although now looking at it closely, I see they've all dropped the e, which obviously doesn't exist in certification.

Up to you if you think it's worth adding these two:
```
certificatin->certification
certificaton->certification
```

Although if you do, the former should probably suggest certificating first.

Reply:
Thanks again. Both seems to be quite common:

https://github.com/search?q=certificatin&type=Code (178 code results)
https://github.com/search?q=certificaton&type=Code (651 code results) 

Updated / added in 305693b9f278a3c9340389e4c944f3e044c02b74